core/src/main/web/app/mainapp/components/publication/publish.jst.html

```diff
@@ -72,6 +72,10 @@ <h1 ng-if="!error">Sign in to Beaker Publications</h1>
             <div class="value update-time">{{model['updated-at'] | date:'short'}}</div>
           </li>
         </ul>
+        <div class="share-private">
+          <label>Share privately</label>
```

Comment:
please label "Unlisted Share" with a tooltip that says "will not be indexed, can only be found if you have the URL"